README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# CSS Homework1
+
+> ## Fernando Alonso Regalado
+
+![Bio Picture](https://github.com/fernandoregalado/hw01/blob/master/picture_bio.jpg)
```

Comment:
According to the link you inserted, I think the way to insert a picture from the local drive is to upload the picture to a folder in GitHub and then insert the link to the markdown. So you taught me how to insert a picture from the local drive.